spec/features/bulk_updates/repopulate_form_from_failed_bulk_updates_spec.rb

```diff
@@ -0,0 +1,468 @@
+require 'rails_helper'
+
+describe 'Repopulating Bulk Update Form after a failed attempt', js: true do
+  before :all do
+    @ingest_response, @concept_response = publish_collection_draft
+  end
+
+  context 'when starting a new bulk update' do
+    before do
+      login
+
+      visit new_bulk_updates_search_path
+
+      # Search form
+      select 'Entry Title', from: 'Search Field'
+      find(:css, "input[id$='query_text']").set(@concept_response.body['EntryTitle'])
```

Comment:
we probably do this all over the bulk update tests, a helper would be useful, but maybe in the next ticket